doc/netplan.md

```diff
@@ -624,6 +624,26 @@ wpasupplicant installed if you let the ``networkd`` renderer handle wifi.
           and ``adhoc`` (peer to peer networks without a central access point).
           ``ap`` is only supported with NetworkManager.
 
+     ``bssid`` (scalar)
+     :    If specified, directs the device to only associate with the given
+          access point.
+
+     ``band`` (scalar)
+     :    Possible bands are ``a`` (for 5GHz 802.11a) and ``bg``
+          (for 2.4GHz 802.11), do not restrict the 802.11 frequency band of the
+          network if unset (the default).
```

Comment:
'a' and 'bg' may be the native terms used in NetworkManager, but I don't think this is a very clear way to label the bands.  I think it's more common to refer to them as 2.4G and 5G, and that these are the terms we should use here explicitly - either 2.4G / 5G or 2.4GHz / 5GHz.  'a' and 'bg' is just going to confuse users who are configuring 802.11ac or 802.11n.

Reply:
I opted for 5GHz/2.4GHz as the documented default, also accepting 5G/2.4G as a fallback.

This is because I want to avoid confusion with future mobile communication protocol standards (3G/4G/5G).